### ISSUE TITLE
system user vagrant config

### ISSUE THREAD

**Issue by plandolt** [vagrant-php/ubuntu#17]
This would provide an easy way to add user specific configuration to every project. I like to have the "joe" editor available on all boxes so I could add a ~/.vagrant.yml (or ~/.vagrant-php/vagrant.yml) file with:

```
general:
    packages:
        - joe
```

**Comment by dominikzogg** [comment]
@scuben this should allready work, as you can see for vim: https://github.com/vagrant-php/ubuntu/blob/14.04/vagrant-default.yml#L116

**Comment by plandolt** [comment]
@dominikzogg Yes I know. For now I add the snippet above to every vagrant-user.yml file in the corresponding project. But I'd like to add it only once to my system user (e.g. ~/.vagrant-php/vagrant.yml).

**Comment by dominikzogg** [comment]
@scuben vagrant-user.yml can be a symlink, which means you can use the same file in each project if you like

**Comment by plandolt** [comment]
@dominikzogg Sure but then one need to add a symlink in every new repository. I added a PR which implements this. With that you can make one file on your system once and then it will always merge into your project.

**Comment by dominikzogg** [comment]
@scuben Vagrantfile is a ruby file, which means we could add a logic which makes an autosymlink if the file ~/vagrant-php/vagrant.yml exists and vagrant-user.yml does not

**Comment by dominikzogg** [comment]
https://github.com/vagrant-php/ubuntu/commit/bd59ec91b61fcc0a996de3b4740a1b66333221c7

**Comment by plandolt** [comment]
:+1:
